$NetBSD: patch-aa,v 1.3 2012/04/22 21:42:42 dholland Exp $

- configure for pkgsrc
- hardwire NetBSD instead of Linux serial port name (XXX: wrong)
- use -O* options that really exist (required by clang)
- don't use -W options that aren't defined for for C++

--- Makefile.orig	2001-04-16 00:56:31.000000000 +0000
+++ Makefile
@@ -34,8 +34,8 @@ CC=g++
 #
 # Pick your YACC processor
 #
-YACC = bison -y
-# YACC = yacc
+# YACC = bison -y
+YACC = yacc
 
 #
 # Define the FLEX processor
@@ -51,11 +51,11 @@ FLEX = flex
 # Use this to define the default device driver name
 # for serial port connections.
 #
-DEFAULT_SERIAL_NAME = "/dev/ttyS0"
+DEFAULT_SERIAL_NAME = "/dev/tty00"
 
 # installation information
-BINDIR=/usr/local/bin
-MANDIR=/usr/local/man/man1
+BINDIR=${PREFIX}/bin
+MANDIR=${PREFIX}/man/man1
 MANEXT=1
 
 
@@ -67,8 +67,8 @@ MV=mv -f
 RM=rm -f
 
 IFLAGS=-Iplatform -Ircxlib -Inqc -Icompiler
-WFLAGS=-Wall -Wstrict-prototypes -Wmissing-prototypes
-CFLAGS = -O6 -pipe $(IFLAGS) $(WFLAGS) -DDEFAULT_SERIAL_NAME='$(DEFAULT_SERIAL_NAME)'
+WFLAGS=-Wall #-Wstrict-prototypes -Wmissing-prototypes
+CFLAGS = -O3 -pipe $(IFLAGS) $(WFLAGS) -DDEFAULT_SERIAL_NAME='$(DEFAULT_SERIAL_NAME)'
 
 OBJ = $(NQCOBJ) $(COBJ) $(RCXOBJ) $(POBJ)
 
